package.json

```diff
@@ -49,7 +49,7 @@
     "open-analyzer": "EXPO_PUBLIC_OPEN_ANALYZER=1 yarn build-web"
   },
   "dependencies": {
-    "@atproto/api": "^0.12.20",
+    "@atproto/api": "0.12.22-next.0",
```

Comment:
Planning to cut a patch or leave as is?

Reply:
Likely can cut beforehand, though this RC was also just cut today so it should be fine.

Anyway I think the PR is in a good state but will ping @devinivy to make sure. Paul should know how to do that. https://github.com/bluesky-social/atproto/pull/2553